financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetNativeModule.kt

```diff
@@ -141,8 +143,20 @@ internal interface FinancialConnectionsSheetNativeModule {
 
         @Singleton
         @Provides
-        fun providesSaveToLinkWithStripeSucceededRepository() = SaveToLinkWithStripeSucceededRepository(
-            CoroutineScope(SupervisorJob() + Dispatchers.Default)
+        fun providesSaveToLinkWithStripeSucceededRepository() =
+            SaveToLinkWithStripeSucceededRepository(
+                CoroutineScope(SupervisorJob() + Dispatchers.Default)
```

Comment:
instead of using default dispatcher could we use the `IOContext` and `UIContext` instead?

Reply:
Good catch, moved to IOContext - https://github.com/stripe/stripe-android/pull/7366/commits/6929aca12333017bf167b4b6ad5454fd1b0f8347

It's a pretty lightweight operation in this repository specifically, but using IOContext as a pattern if that sounds reasonable.